components/list_item/list_item.vue

```diff
@@ -24,6 +25,14 @@
         <!-- @slot named slots for custom list items -->
         <slot :name="slotName" />
       </template>
+      <template
+        v-if="selected"
+        #selected
+      >
+        <dt-icon
+          name="check"
+        />
+      </template>
```

Comment:
Slot for the selected icon that defaults to the check icon?

```suggestion
      <template
        v-if="selected"
        #selected
      >
        <slot name="selected">
          <dt-icon
            name="check"
          />
        </slot>
      </template>
```

Reply:
I agree we should restrict to `<dt-icon name="check" />` — at least to begin with. At some point on Product Design side we'll inventory and standardize on any other visuals ways to show a checked state of a List Item. But yeah, there isn't currently any fundamental reason for it to be a different icon. If there is, that's what the custom option is for.

I'd also love to see examples of it reflected as an example on at least one [Dropdown](https://vue.dialpad.design/?path=/story/components-dropdown--variants)'s nested List items.